cmd/client/util.go

```diff
@@ -57,6 +57,10 @@ func createAPIClient(port string, enableTracing bool) (*clientState, error) {
 }
 
 func deleteAPIClient(cs *clientState) {
+	// TODO: This is to compensate for this bug: https://github.com/grpc/grpc-go/issues/1059
+	//       Remove this delay once that bug is fixed.
+	time.Sleep(time.Duration(50) * time.Millisecond)
```

Comment:
nit: time.Duration is an alias for an int64, this can be simplified to `time.Sleep(50 * time.Millisecond)` and the compiler will handle the type coercion.